.github/scripts/refresh_mirror/refresh-mirror.py

```diff
@@ -0,0 +1,79 @@
+import click
+import time
+import sys
+import backoff
+from azure.devops.connection import Connection
+from msrest.authentication import BasicAuthentication
+
+@click.command()
+@click.argument('pipeline_id', required=True)
+@click.argument('token', required=True)
+@click.option('--commit', default=None)
+@click.option('--cancel', default=None)
+@click.option('--organization', default='https://microsoft.visualstudio.com')
+@click.option('--project', default='HyperVCloud')
+@click.option('--debug', default=False, is_flag=True)
+def main(pipeline_id: str, token: str, commit: str, cancel: str, organization: str, project: str, debug: bool):
+    try:
+        client = Connection(base_url=organization, creds=BasicAuthentication('', token)).clients.get_build_client()
+
+        if cancel is not None:
```

Comment:
I think we can simplify this by making this fire and forget (and not waiting around for the pipeline to finish). Thoughts?

Reply:
I don't imagine we'll be cancelling the mirror pipeline, so removing this logic seems fine. It was more useful for PrivateAdoBuild, while it was still around.